src/EventBuilder.php

```diff
@@ -9,8 +9,7 @@
             $until, // string of date until event will repeat
             $maxRepeats; // integer how much maximum repeats event will occured
     
-    public $dates = [];
-    
```

Comment:
__[code_sniffer]__
* Opening brace of a class must be on the line after the definition